src/huggingface_hub/hf_api.py

```diff
@@ -2308,7 +2257,7 @@ def unlike(
         """
         Unlike a given repo on the Hub (e.g. remove from favorite list).
 
-        See also [`like`] and [`list_liked_repos`].
+        See also [`list_liked_repos`].
```

Comment:
```suggestion
        To prevent spam usage, it is not possible to `like` a repository from a script.

        See also [`list_liked_repos`].
```

I'd add a line here to explain why there isn't a `like` method.